command/pr.go

```diff
@@ -398,7 +398,7 @@ func printPrs(w io.Writer, totalCount int, prs ...api.PullRequest) {
 			prStateColorFunc = utils.Red
 		}
 
-		fmt.Fprintf(w, "  %s  %s %s", prStateColorFunc(prNumber), text.Truncate(50, replaceExcessiveWhitespace(pr.Title)), utils.Cyan("["+pr.HeadLabel()+"]"))
+		fmt.Fprintf(w, "  %s  %s %s ", prStateColorFunc(prNumber), text.Truncate(50, replaceExcessiveWhitespace(pr.Title)), utils.Cyan("["+pr.HeadLabel()+"]"))
```

Comment:
Nit: to avoid unnecessary trailing whitespace, I find it often a good idea to avoid trying to anticipate what comes _after_ a block of text (and adding spaces to "pad" it on the right), and instead focus on every chunk of output ideally having awareness of what _preceded_ it and optionally adding whitespace on the _left_.

This PR changes the approach to what I believe is slightly trickier to maintain (not a huge deal though) and produces trailing spaces at ends of lines in some cases, which is also not a huge deal but I generally advise avoiding it since command-line tools tend to be precise about text output.

<img width="509" alt="Screen Shot 2020-03-20 at 2 17 40 PM" src="https://user-images.githubusercontent.com/887/77167140-b9612e00-6ab5-11ea-93a3-31b7bd0e2928.png">

Reply:
Hey @mislav the reason I tried this was to make sure everything lines up on the left. If we add padding to the left, the pull request title and the checks status are slightly misaligned, which I was thinking hampers the reading experience. 

With the current implementation, things are a bit misaligned: 
```
  #665  switch {pr,issue} view default [preview-default]
   - Checks passing - Approved
```

Do you have suggestions for an implementation that would result in the below kind of formatting? 

```
  #665  switch {pr,issue} view default [preview-default]
  ✓ Checks passing ✓ Approved
```